pipeline/data_merging/bayesdel/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM ubuntu:20.04
+
+RUN apt-get update && apt-get install -y \
+    curl \
+    parallel
+    #&& rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /opt/victor
+
+WORKDIR /opt/victor
+
+RUN curl -L -o VICTOR_linux.tgz https://www.dropbox.com/s/rkfu47k8o4mh5wg/VICTOR_linux.tgz && tar zxf VICTOR_linux.tgz && rm -rf VICTOR_linux.tgz
+
+# TODO split up and do it manually
+
+# installing htslib (includes tabix and bgzip)
+
+WORKDIR /opt
+
+ARG HTSLIB_VERSION=1.10.2
+# TODO squash it together
+RUN curl -L -o htslib-$HTSLIB_VERSION.tar.bz2 https://github.com/samtools/htslib/releases/download/$HTSLIB_VERSION/htslib-$HTSLIB_VERSION.tar.bz2
+
+RUN tar -vxjf htslib-$HTSLIB_VERSION.tar.bz2
+
+RUN apt-get install -y \
+    make \
+    g++ \
+    zlib1g-dev \
+    libbz2-dev \
+    liblzma-dev \
+    libcurl4-nss-dev
+
+RUN cd htslib-$HTSLIB_VERSION && make
+
+RUN cd htslib-$HTSLIB_VERSION && make install
+
+# tabix bgzip parallel
```

Comment:
seems like there's lots of cleanup that can be done here, just removing comments if they're not relevant or explaining why it's worth leaving them there